librarymanagement/src/main/contraband/librarymanagement.json

```diff
@@ -138,19 +138,20 @@
           "namespace": "sbt.librarymanagement",
           "target": "Scala",
           "doc": [
-            "Cross-versions a module using the result of applying `remapVersion` to the binary version.",
-            "For example, if `remapVersion = v => \"2.10\"` and the binary version is \"2.9.2\" or \"2.10\",",
-            "the module is cross-versioned with \"2.10\"."
+            "Cross-versions a module using the result of appending `suffix` to the binary version.",
+            "For example, if `suffix = \"_foo\"` and the binary version is \"2.10\",",
+            "the module is cross-versioned with \"2.10_foo\"."
```

Comment:
You're right. The whole thing is suffixed onto the artifactId (after a `_`). Of course you need the ability to prefix. I'll change it. Thanks for looking.